torchaudio/prototype/ctc_decoder/ctc_decoder.py

```diff
@@ -257,3 +269,48 @@ def lexicon_decoder(
         sil_token=sil_token,
         unk_word=unk_word,
     )
+
+
+def _get_filenames(model: str) -> _PretrainedFiles:
+    files = MODEL_FILE_MAP[model]
+    return _PretrainedFiles(lexicon=files.lexicon, tokens=files.tokens, lm=files.lm if files.lm is not None else None)
```

Comment:
It seems that the re-constructed of `_PretrainedFiles` instance is exactly same as the previous. Is this necessary?